### PR TITLE
fuck you baan

### PR DESCRIPTION
**Pull request by femboy-boop**
kys baan

### CODE REVIEW

**Comment by johnathon-walnut** [comment]
chungy chungy chungus :D

File: Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp [hunk 1]
```diff
@@ -344,7 +340,7 @@ void CAntiAim::Run(CUserCmd* pCmd, bool* pSendPacket) {
 			   }
 			   else 
 			   {
-				  *pSendPacket = bSendReal = false;
+				  *pSendPacket == bSendReal == false;
```
Comment (lnx00) [inline Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp:343]: This line is useless, if you're not using the result of this expression

**Review by lnx00 — COMMENTED** [review]
???

File: Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp (hunk 1 above)
Comment (dox-net) [reply Fedoraware/TeamFortress2/TeamFortress2/Features/AntiHack/AntiAim.cpp:343]: when did i ask 😭

**Pull request merged**